Makefile

```diff
@@ -23,17 +23,21 @@ docs/index.yaml: docs/$(RELEASE).tgz docs/$(RELEASE_LATEST).tgz
 	$(call banner)
 	helm repo index docs --url https://lightbend.github.io/helm-charts
 
-docs/$(RELEASE).tgz: $(CHART)/Chart.yaml $(CHART)/templates/*.yaml
+docs/$(RELEASE).tgz: $(CHART)/* $(CHART)/*/*
```

Comment:
Why these changes?

Reply:
these map all the dependencies properly, so when I twiddle the `es-monitor-api/prometheus.yml` template everything rebuilds properly.

My test loop was basically:

1. change the configs
2. `make install-local-latest` which then makes latest and installs it in minikube
3. run smoketest
4. poke around at the UI